test/govspeak_footnote_test.rb

```diff
@@ -0,0 +1,31 @@
+require "test_helper"
+require "govspeak_test_helper"
+
+class GovspeakFootnoteTest < Minitest::Test
+  include GovspeakTestHelper
+
+  test_given_govspeak "
+    Footnotes can be added[^1].
+
+    [^1]: And then later defined.
```

Comment:
I've one question about this `[^bignote]` [use ](https://www.markdownguide.org/extended-syntax#footnotes) and if this is something that we need to factor in?

Eg
`Here's a simple footnote,[^1] and here's a longer one.[^bignote]`

This doesn't render the same way but then it also doesn't render on govspeak master so prob out of scope.

Reply:
Ooh interesting. I don't think we support or encourage `[^bignote]` notation? 
This is the suggested formatting in whitehall publisher.
So if I understand correctly, seems that we do support multiline footnotes but they should always be numbered. 


<img width="394" alt="Screenshot 2020-11-04 at 14 49 16" src="https://user-images.githubusercontent.com/7116819/98125983-f5ae9680-1eac-11eb-8756-5622593dfda3.png">

Reply:
> I don't think we support or encourage [^bignote] notation?
This is the suggested formatting in whitehall publisher.

Makes sense.